lib/rubycritic/generators/json_report.rb

```diff
@@ -8,7 +8,10 @@ def initialize(analysed_modules)
       end
 
       def generate_report
-        print generator.render
+        FileUtils.mkdir_p(generator.file_directory)
```

Comment:
This would look more complicated, because #mkdir throws error if folder already exists.
`Pathname.new(generator.file_directory).mkdir` unless File.directory? generator.file_directory